onnx/defs/schema.cc

```diff
@@ -79,6 +79,10 @@ OpSchemaRegistry* OpSchemaRegistry::Instance() {
 }
 
 void OpSchema::Verify(const NodeProto& node) const {
+  if (deprecated_) {
+    fail_check("Using a deprecated operator.");
```

Comment:
nit: Maybe say "Operator xxx has been deprecated since version yyy"

Reply:
Done, changed to this and verified:

`fail_check("Operator '", name_, "' has been deprecated since version ", since_version_);
`